pvnet/models/multimodal/multimodal.py

```diff
@@ -176,22 +209,52 @@ def __init__(
             assert pv_history_minutes is not None
 
             self.pv_encoder = pv_encoder(
-                sequence_length=pv_history_minutes // 5 + 1,
```

Comment:
@dfulu This is where I was getting the +1 used below. I can remove all the +1's, having them would include t0 in the inputs.

Reply:
Ah okay, well then maybe ignore my previous comment for now and lets get this merged. Probably something we should come back to though